docs/reference/query-dsl/query-string-syntax.asciidoc

```diff
@@ -271,12 +271,27 @@ of a sub-query:
 If you need to use any of the characters which function as operators in your
 query itself (and not as operators), then you should escape them with
 a leading backslash. For instance, to search for `(1+1)=2`, you would
-need to write your query as `\(1\+1\)\=2`.
+need to write your query as `\(1\+1\)\=2`. In the case of using JSON format for the request body, two preceding backslashes (`\\`) required; since backslash is reserved escaping character in JSON strings.
 
 The reserved characters are:  `+ - = && || > < ! ( ) { } [ ] ^ " ~ * ? : \ /`
```

Comment:
Remove this sentence and relocate below the example.

Reply:
I've committed your changes. Which sentence should I remove?

Reply:
Please remove the sentence at line 276. Thanks!

Reply:
And relocating it below the example? or completely removing it?

Reply:
Completely remove the sentence at line 276

It looks like the same sentence was already added below the example with 28dffd427294aadfcf2c4ba36d4fc5eb6fd1e128. This would remove the duplicate.

Reply:
Oh! My bad.